### ISSUE TITLE
[README Update Request] There should be a section on how to update

### ISSUE THREAD

**Issue by ForceConstant** [rocket-pool/smartnode-install#81]
It seems interesting to me that neither the release notes, or the Readme.md mention how to update smartnode, only install. Can someone add the information about how we should update?

Thanks.

**Comment by jeffersonwarrior** [comment]
Complete guide here: https://docs.rocketpool.net/guides/node/updates.html

**Comment by ForceConstant** [comment]
@jeffersonwarrior yes thanks, and see the specific section regarding updating the smartnode there, but shouldn't there at least be a link to this guide in the readme and/or release notes?
